src/main/java/duke/Deadline.java

```diff
@@ -0,0 +1,38 @@
+package duke;
+
+/**
+ * Special kind of task that has a description and a deadline
+ */
+class Deadline extends Task {
+    protected String type = "D";
```

Comment:
Can consider using the `final` keyword for this variable!

```suggestion
    protected final String type = "D";
```